software/contrib/gate_phaser.md

```diff
@@ -0,0 +1,64 @@
+# Gate Phaser
+
+author: Nik Ansell (github.com/gamecat69)
+
+date: May 2024
+
+labels: sequencer, gates
+
+A script which attempts to answer the question "What would Steve Reich do if he had a EuroPi?".
```

Comment:
I can tell you're very pleased with this phrasing, given the number of times it's used.  But a more descriptive summary of what the program actually does would be helpful as the first-line for documentation.